packages/types-dev/index.d.ts

```diff
@@ -219,6 +219,25 @@ declare global {
             level: LogLevel;
         }
 
+        /** Log message */
+        interface LogObject {
+            /** id of the source instance */
+            from: string;
+            /** log level */
+            severity: string;
+            /** timestamp */
+            ts: number;
+            /** actual content */
+            message: string;
+        }
+
+        interface InternalLogObject extends LogObject {
```

Comment:
should an internal object really be exportet to adapter devs, either we change the name or shouldn't expose it, maybe just call it here already `LogMessage` so the reassignment later is not needed

Reply:
Changed to LogMessage